index.d.ts

```diff
@@ -86,6 +88,8 @@ declare namespace OAuthClient {
         clientSecret: string;
         redirectUri?: string;
         environment?: string;
+        token: Token;
+        logging: boolean;
     }
 
     export enum environment {
```

Comment:
https://github.com/intuit/oauth-jsclient/blob/43a597b946645da558b1670a4a893e0a408a2cda/src/OAuthClient.js#L88
I wasn't sure if constants such as these also had to be included in this file? Lines 79-109 from OAuthClient.js..I did see some constants like the scopes and environment already defined so wanted to clarify